docs/next/components/mdx/MDXComponents.tsx

```diff
@@ -413,6 +418,36 @@ const Code: React.FC<CodeProps> = ({children, dagimage, ...props}) => {
             </svg>
           </div>
         </Transition>
+        {dagimage && (
+          <Transition
+            show={!copied}
+            appear={true}
+            enter="transition ease-out duration-150 transform"
+            enterFrom="opacity-0 scale-95"
+            enterTo="opacity-100 scale-100"
+            leave="transition ease-in duration-150 transform"
+            leaveFrom="opacity-100 scale-100"
+            leaveTo="opacity-0 scale-95"
+          >
+            <div className="absolute top-0 right-6 mt-2 mr-2 block lg:hidden">
+              <svg
+                className="h-5 w-5 text-gray-400 cursor-pointer hover:text-gray-300"
+                viewBox="-1 -1 18 18"
+                fill="none"
+                onClick={toggleDag}
+              >
+                <path
```

Comment:
Minor nit but it might be nice to pull this `<svg>` out to a separate named component since it's difficult to tell from the code what icon is being rendered.